app/assets/stylesheets/v3/layout_components/index.scss

```diff
@@ -45,4 +46,4 @@
 @import "spinner";
 @import "tabs";
 @import "toggle-card";
-@import "data-table";
+@import "toggle-switch";
```

Comment:
I'm not sure if the data-table was intentionally put at the bottom of this cascade? Not sure if you tested this at all.

Reply:
ooohhhhh shoot. haha I was just like "this isn't in alphabetical order" lol I'll just put it back to be safe :)